packages/react-form/src/hooks/submit.ts

```diff
@@ -33,13 +42,14 @@ export function useSubmit<T extends FieldBag>(
     setSubmitting(false);
 
     if (result.status === 'fail') {
-      setRemoteErrors(result.errors);
+      setSubmitErrors(result.errors);
+      propagateErrors(fieldBag, result.errors);
     } else {
-      setRemoteErrors([]);
+      setSubmitErrors([]);
     }
   }
 
-  return {submit, submitting, errors: remoteErrors, setErrors: setRemoteErrors};
+  return {submit, submitting, errors: submitErrors, setErrors: setSubmitErrors};
```

Comment:
so I wonder if the `setErrors` should be a wrapper of both `setSubmitErrors` & `propagateErrors` ? 
```
setSubmitErrors(result.errors);
propagateErrors(fieldBag, result.errors);
```